spec/haml_lint/severity_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+RSpec.describe HamlLint::Severity do
+  let(:name) { nil }
+
+  subject(:severity) { described_class.new(name) }
+
+  it 'defaults to a warning' do
+    subject.should == described_class.new(:warning)
+  end
+
+  it 'can be matched against a symbol severity' do
+    subject.should == :warning
+  end
+
+  it 'ranks severities properly' do
+    subject.should < described_class.new(:error)
+  end
+
+  it 'can wrap itself' do
```

Comment:
This is basically a safeguard against mistakes in passing the severity level. If someone passed a `HamlLint::Severity` where we formerly accepted a symbol severity, it felt like a bad experience to throw an exception. By adding the ability for a `HamlLint::Severity` to "wrap" another, we add an easy way to ensure backward-compatibility between the old symbol-only severity indicators.

This test just ensures that the backward-compatibility doesn't break in the future.